packages/@aws-cdk/aws-apigatewayv2/lib/http/api.ts

```diff
@@ -43,6 +45,13 @@ export interface HttpApiProps {
    * @default - CORS disabled.
    */
   readonly corsPreflight?: CorsPreflightOptions;
+
+  /**
+   * Configure a custom domain with the API mapping resource to the HTTP API
+   *
+   * @default - no default domain mapping configured. meaningless if `createDefaultStage` is `false`.
+   */
+  readonly defaultDomainMapping?: DefaultDomainMappingOptions;
```

Comment:
We should throw an error when `createDefaultStage` is false and `defaultDomainMapping` is specified.

Reply:
OK fixed and added a unit test for it. Will commit later.